### PR TITLE
updated the main to include two data processing steps

### PR DESCRIPTION
**Pull request by dcarver1**
Initial test of our pull review process

- add datasets from email to the project repo 
- attempt to run the 0_main.R 
- note location where the processes failed. 

Function evaluation 
- noise, wildfire : Both are straight forward mean values over an area

Share
- any additional thoughts about the organization of the workflow.

### CODE REVIEW

File: 0_main.R [hunk 1]
```diff
@@ -61,11 +61,14 @@ plan(multisession, workers = 3)
 ## work on what the best integration between the development stratigies is
```
Comment (ccmothes) [inline 0_main.R:61]: On line 52 above, is this setting up for parallel processing in future code versions? Currently the code as is is not set up for parallel processing from what I see... If that was a goal, one thing you could consider using is `furrr` functions in place of `purrr` functions, however I've noticed bugs recently when running functions using terra inside `furrr` functions so might need a different alternative.

**Review by ccmothes — APPROVED** [review]
I have added both inline comments and have some general workflow comments below. I'm marking this as 'Approve' for now but there is a typo in 0_main.Rmd (see in-line note) that needs to be fixed for it to run. I also only reviewed the two additional function files mentioned in the PR description, if there are others that need review let me know in future PRs.

General Workflow:
- For indicator processing functions, consider adding a message at the end of each something like "{indicator} successfully written to {export file path}", I can see that being useful when running this entire workflow and tells users where the files are written to.
- Repetitive code - in at least these two files, the raster extraction functions are identical (and I'm assuming they will be used again). Instead consider create a single separate file/function called "extractValues.R" or something that can be used across indicator-specific functions. The only change needed to make it reproducible is instead of calling out the specific column name in the extracted output, just use "extractedVals[,2]" since the extraction values are always the second column and add an argument for indicator name to name the new column.
- A bit confused by the 'name' argument and use of map2() for both noise and wildfire, since "name" is never used within the processing functions, I would think this should just be a regular map() over each geometry.

**Review by dcarver1 — COMMENTED** [review]
looks good thanks for the commits

**Comment by dcarver1** [comment]
Follow up on comments - 
Repetitive code - I've got this noted 


A bit confused by the 'name' argument and use of map2() - good catch I've alter this

**Pull request merged**